src/Logger/PerMutatorLogger.php

```diff
@@ -79,7 +79,7 @@ private function setUpPerCalculatorMutator(): void
         $processPerMutator = [];
 
         foreach ($processes as $process) {
-            $mutatorName = $process->getMutator()::getName();
+            $mutatorName = $process->getMutatorClass()::getName();
```

Comment:
Does it work since mutator class is a string?

Reply:
Yep